src/facade/redis_parser.cc

```diff
@@ -340,6 +333,15 @@ auto RedisParser::ParseArg(Buffer str) -> Result {
     return BAD_BULKLEN;
   }
 
+  if (c == '_') {  // Resp3 NIL
+    // This is probably a bug?
```

Comment:
what do you mean it's probably a bug?

Reply:
oops, sorry for leaving this like that. I meant that a check for "\r\n" might be missing